src/components/Game.js

```diff
@@ -1,39 +1,79 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Analyzer from './Analyzer.js';
+import Notes from './NoteDetails.js';
+import Info from './Info.js';
 
 class Game extends Component {
   constructor(props){
     super(props);
     this.state={
-      // last_propmted_note: null,
-      // current_prompted_note: null,
-      currentUserNote: null,
+      lastPromptedNoteNum: null,
+      promptedNoteNum: null,
+      promptedNoteLetter: null,
+      promptedNoteFreq: null,
+      userPlayingNote: null,
       // note_matched: false
+      infoShown: false
     }
+  this.toggleInfoShown = this.toggleInfoShown.bind(this);
+  this.increaseSkippedCountCallback = props.increaseSkippedCountCallback.bind(this);
+  }
 
+  toggleInfoShown () {
+    this.setState({ infoShown: !this.state.infoShown });
   }
 
-  getCurrentUserNote = (note) => {
-    this.setState({currentUserNote: note})
-    console.log('in game, currentUserNote: ', this.state.currentUserNote);
+  componentDidMount() {
+    this.setNewNote();
   }
 
-  componentWillUnmount() {
-    if (this.state.processor){this.state.processor.disconnect()};
+  setNewNote() {
+    let noteNum = null;
+    let noteDetails = null;
+    while(noteNum === null || noteDetails === undefined || noteNum === this.state.lastPromptedNoteNum) {
+      noteNum = this.getRandomIntInclusive(16, 42);
+      console.log('noteNum in setNewNote: ', noteNum);
+      noteDetails = this.getNoteDetails(noteNum);
+    }
+
+    this.setState({
+      promptedNoteNum: noteNum,
+      promptedNoteLetter: noteDetails.noteName,
+      promptedNoteFreq: noteDetails.frequency
+    });
+
+    console.log('promptedNoteNum: ', this.state.promptedNoteNum, 'promptedNoteFreq: ', this.state.promptedNoteFreq, 'promptedNoteLetter: ', this.state.promptedNoteLetter);
+  }
+
+  getRandomIntInclusive(min, max) {
+      const randInt = Math.floor(Math.random() * (max - min + 1)) + min;
+      console.log('randInt in getRandomIntInclusive: ', randInt);
```

Comment:
its good to have logging on your dev build but this **should not** flow into production when deployed.

Reply:
I want to ask you about this: how to keep certain things out of deployed code and keep them available for use as you continue to develop. I put it on our list for tomorrow.